tests/unit/amo/components/TestAddonReviewCard.js

```diff
@@ -1071,24 +1076,45 @@ describe(__filename, () => {
     });
 
     it('adds a `src` query parameter to the link in the byLine if available in the location', () => {
```

Comment:
Is this description still accurate? It's not going to add a `src` param with the flag active, is it?

Reply:
ha, it is kinda accurate but I should mention the state of the UTM flag indeed, thanks!

Reply:
Thanks for noticing this one, the test case wasn't updated correctly. I fixed it.